run-zac.sh

```diff
@@ -64,12 +67,15 @@ fi
 fi
 
 if [[ "$1" == "classic" ]]; then
-  echo "Running classic ZAC application"
-  exec node /usr/src/app/server.js classic
+  echo "Running Classic Console"
+  exec node ./server.js classic
 elif [[ "$1" == "edge-api" ]]; then
-  echo "Running ZAC server with Edge API integration"
-  exec node /usr/src/app/server-edge.js
+  echo "Running Console Server with Edge API Integration"
+  exec node ./server-edge.js
```

Comment:
why undo all the full path references for relative references? it doesn't seem to add any value and it often makes it harder to debug if ever needed. i just don't see the value in changing it? Is there some I'm not seeing?

Reply:
I was thinking it'd be useful to be able to exercise the script outside of Docker too, but I reverted the change.